Mission: Ask the Quarg about the Coalition after having joined Heliarchs or Lunarium (#9538)
endless-sky/endless-sky: data/quarg/quarg missions.txt

			`	The Quarg takes one last long look at your ship, then makes a curt farewell motion, leaving you to your business. The crowd, too, starts to disperse, with only a few staying behind to gawk at the ship.`
			`	As you make your way through the ring, you notice occasional glares from the Quarg onboard the ring.`
				decline

mission "Ask Quarg About Coalition Late"
	invisible
	landing
	deadline 1
	to offer
		has "Ask Quarg About Coalition Early: offered"
		or
			has "joined the heliarchs"
			has "joined the lunarium"
	source
		attributes "quarg"
		attributes "station"
	on offer
		conversation
			branch heliarch
				has "joined the heliarchs"
			branch lunarium
				has "joined the lunarium"
			label heliarch

> 			`You've obtained the rank of Heliarch, the group who to this day remain hostile to the Quarg. Despite that you were allowed to land here on <origin> - you don't know whether that's because the Quarg simply don't know that you've joined the Heliarchs, or don't care enough to stop you from coming to their rings. Would you like to ask the Quarg for their side of the story, in light of what you've learned from the Heliarchs?`

Reviewer: ```suggestion
			`You've obtained the rank of Heliarch, the group who to this day remain hostile to the Quarg. Despite that, you were allowed to land here on <origin> - you don't know whether that's because the Quarg simply don't know that you've joined the Heliarchs, or don't care enough to stop you from coming to their rings. Would you like to ask the Quarg for their side of the story, in light of what you've learned from the Heliarchs?`
```

			`You've joined the Lunarium in Coalition space, and even served as a messenger to bring the Quarg some information about the Lunarium itself, in the hopes that they might provide some help. Would you like to ask the Quarg for their side of the story, in light of what you've learned from the Lunarium and from your visit to the Quarg with Oobat?`
			choice
				`	(Yes.)`
					goto yes
				`	(Not now.)`
					defer
				`	(No, I'm not interested.)`
					decline
			label yes
			`	You're never quite sure who to speak with when it comes to the Quarg, but you're instinctively drawn to one that stands out. You judge it to be an older member of their species; its thin scales cover its fingers, and the more bony parts of its head have turned to a bright, silvery shade. Unlike most Quarg you've seen, it is wearing a pale, faded white robe, with patches of red-colored chains (that somehow perfectly transition into fabric again) becoming more and more prominent closer to the top, culminating in a firm, crimson hood covering its head. Most surprising of all for a Quarg, however, is the large scar taking up much of the left side of its face, completely covering where its left eye should be.`

> 			`	It is completely static, with a lost gaze focused on the ceiling here until you approach it. It doesn't squat down to meet your eye level, and says nothing, simply gazing at you with its one eye, colored a bright gold.`

Reviewer 2: ```suggestion
			`	It is completely static, with a lost gaze focused on the ceiling here until you approach it. It doesn't squat down to meet your eye level. Instead it says nothing, simply gazing at you with its one eye, colored a bright gold.`
```

> 			`	It is completely static, with a lost gaze focused on the ceiling here until you approach it. It doesn't squat down to meet your eye level. Instead it says nothing, simply gazing at you with its one eye, colored a bright gold.`

Reviewer 3: ```suggestion
			`	It is completely static, with a lost gaze focused on the ceiling until you approach it. It doesn't squat down to meet your eye level. Instead it says nothing, simply gazing at you with its one eye, colored a bright gold.`
```

			choice
				`	"Hello, I'm <first> <last>. What can you tell me about the Coalition?"`
				`	"Hello. Is it true that the Heliarchs defeated you and stole your ringworlds?"`

> 			`	It continues to stare at you for a while, without as much as blinking. It looks over to a bench in one of the quieter areas here, and gestures for you to follow. As you two sit down, it finally speaks up.`

Reviewer 4: ```suggestion
			`	It continues to stare at you for a while, without as much as blinking. It looks over to a bench in one of the quieter areas here, and gestures for you to follow. As you two sit down it finally speaks up.`
```
These three sentences all have the same structure, which looks off. The answer is, of course, commas.

> 			`	It continues to stare at you for a while, without as much as blinking. It looks over to a bench in one of the quieter areas here, and gestures for you to follow. As you two sit down it finally speaks up.`

Reviewer 3: ```suggestion
			`	It continues to stare at you for a while, without as much as blinking. It looks over to a bench in one of the quieter areas of the spaceport, and gestures for you to follow. As you two sit down it finally speaks up.`
```

			`	"Those three species arose in relative rapid succession, all in close proximity, compared to how other species tend to be separated by long periods of time, and longer still their distances. So large are the gaps, that one might never learn of those that came before. But, that was not the case for the three. The first of them, the Saryds, were in many ways much like us Quarg in our younger days, and we perhaps were much too lenient because of that... The Kimek, traumatized by the billions fallen in their famines, sought help to make sure they would never face such a crisis again, and were eager to establish friendly relations with the Saryds. The Arachi, in a way, parallel humanity's development, though they were less keen on letting their inner conflicts be made public. They were the last of the three, and upon meeting their two seniors, invested heavily in research to catch up.`

> 			`	"In the intersecting point of their territories, we built our ringworlds. Seven thousand Earth years ago, the three species of the Coalition sought to take over those rings. After centuries... no, a millennium of fighting, we left that part of the galaxy."`

Reviewer 3: We've described this Quarg in great detail, then have this action where the Quarg decides to only speak to you in a specific section of the spaceport, but then we immediately jump into lore dumping. I feel like we're missing a description of how the Quarg is speaking. Is there there a certain tone in its voice that we can pick up on? Are there any discernable facial expressions that it's making as it speaks? Is it showing emotion through its scales like other Quarg? (Can it not as you mentioned on stream, and that be something the player character picks up on?) Something to break up the lore dump a little.

			choice
				`	"Why did they decide to start attacking you and your rings?"`
				`	"Even now you're more advanced than them, aren't you? How come you lost thousands of years ago, when they were weaker?"`
					goto lost

> 			`	"Whatever reasons they had, whatever drove them to attack us, we cannot quite comprehend..." It stops, not sighing or showing much emotion. It looks at the ceiling again, then back at you. "No. You are tired of hearing this same story from us Quarg, are you not? That the foolish and young species warred for no good reason," it pauses again, contemplating its words. "That war was not the first instance where we lost a ring of ours, but never did any of us wish for any ring to be lost in the first place. With three rings there, with the living space for over 3 trillion Quarg being constructed, we got awfully... protective of that region, in order to to prevent outside interference from causing us trouble again. Our relations with the Kimek and Arachi were a far cry from the initial success and amicability we enjoyed with the Saryds, and when an Arach ship first showed signs of technology that repaired its hull platings mid-flight, an achievement even the Saryds were yet to make, we... jumped to conclusions."`

Reviewer 5: ```suggestion
			`	"Whatever reasons they had, whatever drove them to attack us, we cannot quite comprehend..." It stops, not sighing or showing much emotion. It looks at the ceiling again, then back at you. "No. You are tired of hearing this same story from us Quarg, are you not? That the foolish and young species warred for no good reason," it pauses again, contemplating its words. "That war was not the first instance where we lost a ring of ours, but never did any of us wish for any ring to be lost in the first place. With three rings there, with the living space for over three trillion Quarg being constructed, we got awfully... protective of that region, in order to to prevent outside interference from causing us trouble again. Our relations with the Kimek and Arachi were a far cry from the initial success and amicability we enjoyed with the Saryds, and when an Arach ship first showed signs of technology that repaired its hull platings mid-flight, an achievement even the Saryds were yet to make, we... jumped to conclusions."`
```

			`	It once again starts looking at the ceiling, remaining like that for a minute or so.`
			branch "heliarch 2"
				has "joined the heliarchs"
			branch "lunarium 2"
				has "joined the lunarium"
			label "heliarch 2"
			`	It then turns to face you once more. "We have of course noticed that headpiece the Heliarchs have provided you with. Though you have officially joined their ranks, you are human. You are not bound to that area of space. There is no need for you to obey their orders. You do not need to ever return there. There is no need for you and us Quarg to be enemies."`
			label "heliarch ask"
			branch end
				and
					has "ask quarg coalition late: threat"
					has "ask quarg coalition late: aid"
			choice
				`	"Is that a threat?"`
					goto threat
						to display
							not "ask quarg coalition late: threat"
				`	"What if I do want to aid the Heliarchs?"`
					goto aid
						to display
							not "ask quarg coalition late: aid"
			label threat
			action
				set "ask quarg coalition late: threat"

> 			`	"No. It is advice. Though you have joined them, you are still welcome at our rings, for the time being. But, the Heliarchs are a group formed and bound by their distrust and repulse for our kind. Should you continue to follow the Heliarchs, sooner or later they are sure to put you against us, at which point I can no longer guarantee you will retain safe passage through our systems, or will be granted landing clearance on our worlds still."`

Reviewer 3: ```suggestion
			`	"No. It is advice. Though you have joined them, you are still welcome at our rings, for the time being. But, the Heliarchs are a group formed and bound by their distrust and revulsion for our kind. Should you continue to follow the Heliarchs, sooner or later they are sure to put you against us, at which point I can no longer guarantee you will retain safe passage through our systems, or will be granted landing clearance on our worlds still."`
```

			label "lunarium ask"
			branch end
				and
					has "ask quarg coalition late: help"
					has "ask quarg coalition late: tips"
			choice
				`	"Will the Quarg help us fight the Heliarchs?"`
					goto help
						to display
							not "ask quarg coalition late: help"
				`	"Any tips on how to fight them?"`
					goto tips
						to display
							not "ask quarg coalition late: tips"
			label help
			action
				set "ask quarg coalition late: help"
			`	"We agreed to that conversation, that 'interview' of yours. Beyond that, we cannot help much without provoking war, I fear. We seek not to strike fear into the hearts of those of the Coalition, we never did. It would burden your cause to have us take part in the fighting."`
				goto "lunarium ask"
			label tips
			action
				set "ask quarg coalition late: tips"
			`	"Do not be arrogant. Do not underestimate them. Do not take victory for granted, no matter how cornered they may seem, lest cornered yourself you shall be."`
				goto "lunarium ask"
			label end
			action

> 				clear "ask quarg coalition late: help
> 				clear "ask quarg coalition late: tips
> 				clear "ask quarg coalition late: threat
> 				clear "ask quarg coalition late: aid

Reviewer 6: ```suggestion
				clear "ask quarg coalition late: help"
				clear "ask quarg coalition late: tips"
				clear "ask quarg coalition late: threat"
				clear "ask quarg coalition late: aid"
```

Author: Was fixing after I saw the failed check, missed your suggestion here lol

			`	You get up and are just about to leave when it says, "<first> <last>." The Quarg's gaze pierces you, its eye on the same level as your own. "Where is your home?"`
			choice
				`	"I come from New Boston, on the Dirt Belt."`
					goto planet
				`	"My ship is my home."`
					goto ship
				`	"I don't really have a home."`
			`	Your response seems to have struck something in it, as it brings one hand over its scar, glancing downwards. "A harrowing sensation, is it not?"`
				goto name
			label planet

> 			`	It acknowledges your answer with the faintest of nods. "Sparse were my own visits to the rocky worlds inhabited by other species. They are unlike our own. So different, so... complete." its eye darts around, looking at your surroundings, other Quarg here, then finally its own hands and arms. "But us Quarg must make do."`

Reviewer 3: ```suggestion
			`	It acknowledges your answer with the faintest of nods. "Sparse were my own visits to the rocky worlds inhabited by other species. They are unlike our own. So different, so... complete." Its eye darts around, looking at your surroundings, other Quarg here, then finally its own hands and arms. "But us Quarg must make do."`
```

				goto name
			label ship

> 			`	After staring you down for a few seconds, it fixates upon the ceiling again. "Similar to our own lives in our ringworlds, I suppose others such as you may imagine that it is, to live within a ship, or simple space station." It pauses again, for nearly a minute this time. "Considering the circumstances, a reasonable conclusion on your part, that may be."`

Reviewer 2: ```suggestion
			`	After staring you down for a few seconds, it fixates upon the ceiling again. "I suppose others such as you may imagine that our own lives in our ringworlds are similar to your own within a ship or simple space station." It pauses again, for nearly a minute this time. "Considering the circumstances, a reasonable conclusion on your part, that may be."`
```

			label name
			`	It faces you again, and asks, "Have you any more questions for me, Captain?"`
			choice
				`	"Who are you?"`
			`	"I am called Gohl Salyi Memt-Wrik Laz. An old Quarg. Too old for my own good."`
			`	You thank it for its time, and see it get up to leave. As soon as it steps away from the bench, you notice a dozen heavily armored Quarg begin to follow it, some coming from the sides of the bench, others seemingly having materalized from the very walls and flooring. They follow Gohl in a machine-like formation, and never so much as acknowledge you.`
			`	When you last glance back at it before entering your ship again, you see the distinct reflection of Gohl's eye glancing at you from a distance, before being followed by the armorclad Quarg deeper into the station.`
				accept
	on accept

> 		log "People" "Gohl" `Full name Gohl Salyi Memt-Wrik Laz, it is an old, odd Quarg that is more open than its species tends to be in answering questions about rather sensitive topics.`

Reviewer 2: ```suggestion
		log "People" "Gohl" `Full name Gohl Salyi Memt-Wrik Laz. It is an old, odd Quarg that is more open than its species tends to be in answering questions about rather sensitive topics.`
```
